yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -427,23 +422,24 @@ public class Document(public val key: Key) {
                 public data class PresenceChanged(public val changed: PresenceInfo) : MyPresence
             }
 
-            public sealed interface Others : PresenceChange {
+            public sealed class Others(public open val changed: PresenceInfo) : PresenceChange {
 
                 /**
                  * Means that the client has established a connection with the server,
                  * enabling real-time synchronization.
                  */
-                public data class Watched(public val watched: PresenceInfo) : Others
+                public data class Watched(override val changed: PresenceInfo) : Others(changed)
 
                 /**
                  * Means that the client has been disconnected.
                  */
-                public data class Unwatched(public val unwatched: PresenceInfo) : Others
+                public data class Unwatched(override val changed: PresenceInfo) : Others(changed)
 
                 /**
                  * Means that the presences of the client has been updated.
                  */
-                public data class PresenceChanged(public val changed: PresenceInfo) : Others
+                public data class PresenceChanged(override val changed: PresenceInfo) :
+                    Others(changed)
             }
```

Comment:
I think this would be more compact.
```
            public sealed interface Others : PresenceChange {
                public val changed: PresenceInfo

                /**
                 * Means that the client has established a connection with the server,
                 * enabling real-time synchronization.
                 */
                public data class Watched(override val changed: PresenceInfo) : Others

                /**
                 * Means that the client has been disconnected.
                 */
                public data class Unwatched(override val changed: PresenceInfo) : Others

                /**
                 * Means that the presences of the client has been updated.
                 */
                public data class PresenceChanged(override val changed: PresenceInfo) : Others
            }
```

Reply:
I changed `Others` back to interface in [9ae165c](https://github.com/yorkie-team/yorkie-android-sdk/pull/135/commits/9ae165c8e01009ceeb35687092f6cb4c094e919c).